README.md

```diff
@@ -67,6 +69,8 @@ module.exports = withMdxEnhanced({
 })(/* your normal nextjs config */)
 ```
 
+*This plugin supports placing pages under `src/pages` [as documented here](https://nextjs.org/docs/advanced-features/src-directory)*
+
```

Comment:
It seems like you have moved towards trying to internally automatically detect whether `src` is used, rather than passing an explicit option. If this is the case, we probably don't even need this note - it will just work.